modules/nf-core/rtgtools/svdecompose/main.nf

```diff
@@ -0,0 +1,50 @@
+process RTGTOOLS_SVDECOMPOSE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/rtg-tools:3.12.1--hdfd78af_0':
+        'biocontainers/rtg-tools:3.12.1--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(vcf), path(tbi)
+
+    output:
+    tuple val(meta), path("*.vcf.gz"), emit: bnd_vcf
```

Comment:
Why is the channel not just vcf?

Reply:
This VCF only involves BND type of variants.